### PR TITLE
Flake8 and Black

### PR DESCRIPTION
**Pull request by marcuspp**

### CODE REVIEW

File: app/algorithm/sql_statements.py [hunk 1]
```diff
@@ -1,34 +1,49 @@
...
+    CompaniesHouseIDMapping,
+    CompanyDescriptionModel,
+    CompanyNameMapping,
+    ContactEmailMapping,
+    DunsNumberMapping,
+    PostcodeMapping,
+)
 
 
-_ch_name_simplification = """
```
Comment (dvoong) [inline app/algorithm/sql_statements.py]: the original is more readable in my opinion

File: app/algorithm/sql_statements.py [hunk 2]
```diff
@@ -1,34 +1,49 @@
...
+    "regexp_replace("
+    "company_name,"
+    "'^the\\s|\\s?:\\s?|\\[|\\]|\\(|\\)|\'\'|\\*.*\\*|&|,"
+    "|;|\"|ltd\\.?$|limited\\.?$|\\sllp\\.?$|\\splc\\.?"
+    "$|\\sllc\\.?$|\\sand\\s|\\sco[\\.|\\s]|\\scompany[\\s|$]', ' ', 'gi'),"
+    "'\\.|\\s', '', 'gi')"
+    ", ''), company_name))"
+)
 
-_general_name_simplification = f"""
```
Comment (dvoong) [inline app/algorithm/sql_statements.py]: here too

File: app/algorithm/sql_statements.py (hunk 1 above)
Comment (marcuspp) [reply app/algorithm/sql_statements.py]: Yeah this was my area of concern. Python requires `\\` double slashes and trying to keep it within the 100 char limit. Plus black complains about indentation if i left the sql indented. 

We could exclude this file from the checks (which i think i possible)?

Comment (marcuspp) [reply app/algorithm/sql_statements.py]: I've reverted these two blocks and excluded them from the blocks

**Pull request merged**